apps/backoffice-v2/src/pages/Statistics/components/WorkflowStats/WorkflowStats.tsx

```diff
@@ -0,0 +1,325 @@
+      name: 'Pending manual review',
+      value: 15,
+    },
+    {
+      id: 'ckl1y5e0x0002wxrmnd8j9rb7',
+      name: 'Collection Flow',
+      value: 5,
+    },
+    {
+      id: 'ckl1y5e0x0002wxrmnd8j9rb7',
+      name: 'Revisions',
+      value: 3,
+    },
+    {
+      id: 'ckl1y5e0x0002wxrmnd8j9rb7',
+      name: 'Awaiting 3rd party data',
+      value: 2,
+    },
+  ];
+  const tagsWithColors = useMemo(
+    () =>
+      tags
+        ?.slice()
+        ?.sort((a, b) => b.value - a.value)
+        ?.map((filter, index) => ({
+          ...filter,
+          color: HSL_PIE_COLORS[index],
+        })),
+    [tags],
+  );
+  const assignedTags = useMemo(
+    () => tagsWithColors?.reduce((acc, filter) => acc + filter.value, 0),
+    [tagsWithColors],
+  );
+  const visibleActiveCasesAmount = 4;
+  const visibleActiveCases = useMemo(
+    () => tagsWithColors?.slice(0, visibleActiveCasesAmount),
+    [tagsWithColors],
+  );
+
+  const filtersPendingManualReview = [
+    {
+      id: 'ckl1y5e0x0000wxrmsgft7bf0',
+      name: 'Merchant Onboarding',
+      value: 15,
+    },
+  ];
+  const filtersPendingManualReviewWithColors = useMemo(
+    () =>
+      filtersPendingManualReview
+        ?.slice()
+        ?.sort((a, b) => b.value - a.value)
+        ?.map((filter, index) => ({
+          ...filter,
+          color: HSL_PIE_COLORS[index],
+        })),
+    [filtersPendingManualReview],
+  );
+  const assignedCasesPendingManualReview = useMemo(
+    () => filtersPendingManualReviewWithColors?.reduce((acc, filter) => acc + filter.value, 0),
+    [filtersPendingManualReviewWithColors],
+  );
+  const visibleCasesPendingManualReviewAmount = 4;
+  const visibleCasesPendingManualReview = useMemo(
+    () => filtersPendingManualReviewWithColors?.slice(0, visibleActiveCasesAmount),
+    [filtersPendingManualReviewWithColors],
+  );
+  const assignees = [
+    {
+      name: 'John Doe',
+      value: 3,
+    },
+    {
+      name: 'Jane Doe',
+      value: 8,
+    },
+    {
+      name: 'Bob Smith',
+      value: 0,
+    },
+  ];
+  const CustomLegend: ComponentProps<typeof Legend>['content'] = ({ payload }) => (
+    <div className={'mb-6 flex items-end'}>
+      <ul className="ms-auto">
+        {payload?.map((entry, index) => (
+          <li key={`item-${index}`} className="flex items-center gap-x-2">
+            <span
+              className="mt-1 flex h-2 w-2 rounded-full"
+              style={{
+                backgroundColor: `rgb(0, 122, 255)`,
+              }}
+            />
+            {titleCase(entry.value ?? '')}
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+  const resolvedCasesByMonth = [
+    { month: 'Jan 22', category1: 650 },
+    { month: 'Feb 22', category1: 1300 },
+    { month: 'Mar 22', category1: 2600 },
+    { month: 'Apr 22', category1: 300 },
+    { month: 'May 22', category1: 1300 },
+    { month: 'Jun 22', category1: 1950 },
+    { month: 'Jul 22', category1: 100 },
+  ];
+
+  return (
```

Comment:
Same. Plenty of blocks here, it will make maintainability really hard. Please consider splitting it.